app/test/e2e/utils.js

```diff
@@ -54,6 +54,211 @@ const createResource = (app = 'rw', vocabularyCount = 1) => {
     };
 };
 
+/**
+ * Mocks the requests for validating a dataset, returning the id of the mock dataset.
+ *
+ * @param {Object} opts Options object
+ * Should contain the nock instance, and (optional) an id to use as the mock dataset's id.
+ *
+ * @returns {String} The id of the mock dataset.
+ */
+const mockDataset = ({ nock, id = undefined }) => {
```

Comment:
why pass a library as argument? just `const nock = require('nock')` in this file.

Reply:
also: function in js support multiple arguments :P while I see the added clarity of "faking" named arguments, that also has the drawback of making it possible to forget an argument. It's also something that's not used anywhere else on the project, so I think we shouldn't use it

Reply:
this is the most I ever commented on a single line, but here's a 3rd suggestion: https://github.com/resource-watch/dataset/blob/08f08424362866dc8eb5c479b958ad93afc88c7c/app/test/e2e/utils.js#L91

instead of accepting a single param, and having to add more and more as we reuse this code in the future, expect an object, and merge that with the dataset down below. Makes this a lot more flexible.

Reply:
1. My bad, now importing nock inside the utils helper.
2. Keeping the consistency across the project, removed named arguments usage :)
3. Done also, thanks for the suggestion!